src/services/middlewareServices/AuthorizationMiddlewareService.ts

```diff
@@ -0,0 +1,63 @@
+import { NotFoundError } from "@errors/NotFoundError"
+
+import SessionData from "@classes/SessionData"
+
+import AuthService from "@services/identity/AuthService"
+import IsomerAdminsService from "@services/identity/IsomerAdminsService"
+import UsersService from "@services/identity/UsersService"
+
+import { E2E_ISOMER_ID } from "./AuthMiddlewareService"
+
+// Import logger
+const logger = require("@logger/logger")
+
+interface AuthorizationMiddlewareServiceProps {
+  identityAuthService: AuthService
+  usersService: UsersService
+  isomerAdminsService: IsomerAdminsService
+}
+
+export default class AuthorizationMiddlewareService {
```

Comment:
we should change the name of this service to differentiate it from `authMiddlewareService`

Reply:
https://github.com/isomerpages/isomercms-backend/pull/479/commits/e5867960974558b7ddc0593eb1c2bbc07c9a50bc